packages/editor/src/components/post-permalink/style.scss

```diff
@@ -36,6 +36,15 @@
 		// buttons to shrink anyway.
 		flex-shrink: 0;
 	}
+
+	// Hide the thick block border in the Code Editor.
+	.edit-post-text-editor__body .editor-post-title & {
```

Comment:
We're in the editor module, we should reference edit-post styles. If we were to override this, it should happen in the `edit-post` module where we use the post title component.